doc/subsystems/nvs/nvs.rst

```diff
@@ -90,9 +97,14 @@ More generally then, with
 - ``SECTOR_SIZE`` in bytes, and
 - ``PAGE_ERASES`` as the number of times the page can be erased,
 
-the expected device life (in minutes) can be calculated as::
+for DS < 128 byte, the expected device life (in minutes) can be calculated as::
+
+   ``SECTOR_COUNT * (SECTOR_SIZE-8) * PAGE_ERASES / (NS * (DS+4)) minutes``
+
+for DS >= 128 byte, the expected device life (in minutes) can be calculated
+as::
 
-   SECTOR_COUNT * (SECTOR_SIZE-8) * PAGE_ERASES / (NS * (DS+8)) minutes
+   ``SECTOR_COUNT * (SECTOR_SIZE-8) * PAGE_ERASES / (NS * (DS+8)) minutes``
```

Comment:
remove the double back ticks here.  The double colon at the end of the previous line says the following indented text is a code-block, so as written the back ticks will show up in the generated doc output.